### PR TITLE
Repo Migration - Replaced all references to timehop by adsbynimbus

### PR DESCRIPTION
**Pull request by brugg**

### CODE REVIEW

**Review by inderdhir — APPROVED** [review]

File: com.adsbynimbus.nimbus/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   ],
   "author": {
     "name": "Marc Santiago",
-    "email": "marc@timehop.com",
+    "email": "marc@adsbynimbus.com",
```
Comment (marcsantiago) [inline com.adsbynimbus.nimbus/package.json:14]: should marc@nimbus.co

**Comment by marcsantiago** [comment]
This package hasn't been migrated to the adsbynimbus org...  You migrate first AND then update references.

**Review by marcsantiago — CHANGES_REQUESTED** [review]
This package hasn't been migrated to the adsbynimbus org... You migrate first AND then update references typically. For Likewise, moving to the new reference means an update in Unity installation docs are needed.

File: com.adsbynimbus.nimbus/package.json (hunk 1 above)
Comment (brugg) [reply com.adsbynimbus.nimbus/package.json:14]: True. Thanks

**Review by marcsantiago — APPROVED** [review]

**Pull request merged**